### DISCUSSION TITLE
Block selection

### DISCUSSION THREAD

**Discussion by GyorgySZEIDL** [texstudio-org/texstudio#3573] [Q&A]
Block selection does not work properly under texstudio 4.7.3. 
The command 

Shift+Ctrl+Mouse: Selects a block

does not select a vertical block in the itemize environment.  What might be the reason 
of the misbehavior? (gyorgy.szeidl@gmail.com)

**Comment by sunderme** [comment]
not sure why this is in "discussions" (please open an issue)
It works fine here, so "misbehaviour" needs to be explained in more detail.

**Reply by GyorgySZEIDL** [reply]
Compare the result under texstudio and winedt11. The blue area show the results obtained 
when I pressed Shift+Ctrl+Mouse: and made an attempt to select a block using texstudio and
winedt11. 
![BlockSelection](https://github.com/texstudio-org/texstudio/assets/37615412/6a92f8d0-16a7-4bf5-8c37-1c2dc0d38d65)

**Comment by sunderme** [comment]
txs looks at complete lines and makes a block selection over that.
Your example shows soft-wrapping/virtual line breaks and apparently winedt makes block selection within *one* line which is wrapped.
I can't think of a pratical use-case where that makes sense.
The application for block select is something like a multi-row table, every row in a new line, and select columns this way.

So, txs is fine, winedt behaviour is unexpected and will not be replicated in txs.

**Comment by foreachthing** [comment]
My opinion is, that txs is the odd ball here. Notepad++ and VSCode do it the same way as winedt11 - the expected way, when selecting a _block_.

**Comment by sunderme** [comment]
vs code does not wrap lines, and as far as I am aware, neither notepad++
